src/example/example.service.ts

```diff
@@ -0,0 +1,93 @@
+export class ExampleService {
+  public calculateAverage(numbers: number[]): number {
+    if (!numbers.length) return 0;
+    let sum = 0;
+    for (const num of numbers) {
+      sum += num;
+    }
+    return sum / numbers.length;
```

Comment:
PullProofAI

The `calculateAverage` method calculates the average of numbers in an array. Consider using the `reduce` method for a more concise and functional approach.

```typescript
public calculateAverage(numbers: number[]): number {
    if (!numbers.length) return 0;
    const sum = numbers.reduce((acc, num) => acc + num, 0);
    return sum / numbers.length;
}
```



<!-- This is an auto-generated comment by PullProofAI -->